index.js

```diff
@@ -11,3 +11,49 @@ document.querySelectorAll('.navbar-tools').forEach((n) => n
     hamburger.classList.remove('active');
     desktopNav.classList.remove('active');
   }));
+
+const form = document.getElementById('form');
+
+const email = document.getElementById('email');
+
+function isEmail(email) {
+  if (email.toLowerCase() === email) {
+    return true;
+  }
+  return false;
+}
+
+function checkInputs(event) {
+  // get the value from inputs
+  const emailValue = email.value;
+
+  if (isEmail(emailValue)) {
+    return true;
+  }
+
+  // setErrorFor('Invalid Email, it should only contain lowercase');
```

Comment:
- [ ] Kindly remove the commented code from the codebase to make your code clean and follow JavaScript best practices.

Reply:
Done, Thank you.